airflow/cli/commands/internal_api_command.py

```diff
@@ -240,11 +241,11 @@ def create_app(config=None, testing=False):
 
     with flask_app.app_context():
         init_error_handlers(flask_app)
-        init_api_internal(flask_app, standalone_api=True)
+        init_api_internal(connexion_app, standalone_api=True)
 
         init_jinja_globals(flask_app)
         init_xframe_protection(flask_app)
-    return flask_app
+    return connexion_app
```

Comment:
Why not returning the `flask_app` instead of the `connexion_app`? That would avoid making many changes around this

Reply:
Thanks, got it. here `connexion_app.app` returns a `FlaskApp` so we can return it. Can you explain or give some reference on the benefits you anticipate? We do not have the full context of this PR yet. Will help to understand the migration better 💯

Reply:
The will decrease the number of changes. There are many changes in this PR which replace `cached_app(config={"FAB_UPDATE_PERMS": False}).appbuilder` by `cached_app(config={"FAB_UPDATE_PERMS": False}).app.appbuilder`. By doing so, we will no longer need these changes

Reply:
Thanks! Got it 🚀